tweakwcs/tests/test_linearfit.py

```diff
@@ -25,9 +26,12 @@
     'general': linearfit.fit_general,
 }
 
-_ATOL = 10 * _LARGE_SAMPLE_SIZE * np.sqrt(
-    np.finfo(linalg._MAX_LINALG_TYPE).eps
-)
+if linalg._MAX_LINALG_TYPE is not None:
+    _ATOL = 10 * _LARGE_SAMPLE_SIZE * np.sqrt(
+        np.finfo(linalg._MAX_LINALG_TYPE).eps
+    )
+else:
+    _ATOL = 10 * _LARGE_SAMPLE_SIZE * sys.float_info.epsilon
```

Comment:
Should the `sys.float_info.epsilon` include a `np.sqrt`? Also, why not `np.float64` which appears to be the currently used value which leads to this warning:
```
>> np.finfo(None)
<ipython-input-2-7bf5390f0d08>:1: DeprecationWarning: finfo() dtype cannot be None. This behavior will raise an error in the future. (Deprecated in NumPy 1.25)
  np.finfo(None)
Out[2]: finfo(resolution=1e-15, min=-1.7976931348623157e+308, max=1.7976931348623157e+308, dtype=float64)
```
I must admit I don't understand the justification for this selection of a `_ATOL` value so I'm happy to defer to your change.